easybuild/easyconfigs/s/Sambamba/Sambamba-0.7.1.eb

```diff
@@ -0,0 +1,38 @@
+# Version: 0.6.6
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+#
+# Version: 0.7.1
+# Author: Alex Domingo
+# SISC - Vrije Universiteit Brussel
+
+easyblock = 'PackedBinary'
+
+name = 'Sambamba'
+version = '0.7.1'
+local_distbin = '%(namelower)s-%(version)s-linux-static'
+
+homepage = 'https://lomereiter.github.io/sambamba/'
+description = """Sambamba is a high performance modern robust and fast tool
+ (and library), written in the D programming language, for working with SAM
+ and BAM files. Current functionality is an important subset of samtools
+ functionality, including view, index, sort, markdup, and depth."""
+
+toolchain = SYSTEM
+
+source_urls = ['https://github.com/biod/%(namelower)s/releases/download/v%(version)s']
+sources = ['%s.gz' % local_distbin]
+checksums = ['d6e3faaa4a5440067e65b396c9a23188765ea5acc6621485c48ac86158fa62f3']
+
+postinstallcmds = [
+    'chmod 755 %%(installdir)s/%s' % local_distbin,
+    'cd %%(installdir)s && ln -s %s %%(namelower)s' % local_distbin,
+]
+
+sanity_check_paths = {
+    'files': ['%(namelower)s', '%s' % local_distbin],
```

Comment:
@lexming nit: this can be simplified to:
```
    'files': ['%(namelower)s', local_distbin],
```

Reply:
@lexming sorry for committing this change myself, I wanted to make a PR for it

Reply:
@smoors no problem at all, I actually prefer that maintainers do direct commits than starting an inception of PRs :smiley: